src/test/java/study/racing/domain/CarsTest.java

```diff
@@ -0,0 +1,39 @@
+package study.racing.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class CarsTest {
+
+    @Test
+    @DisplayName("자동차들에게 움직이지 않도록 할 수 있다..")
+    void doNotMoveCarsTest() {
+        // given
+        Cars cars = new Cars(new CarsNumber("2"));
+        DoNotMoveGenerator doNotMoveGenerator = new DoNotMoveGenerator();
+
+        // when
+        cars.move(doNotMoveGenerator, new TryCounts("1"));
```

Comment:
```suggestion
        List<Integer> moveNumbers = List.of(3,2);
        cars.move(moveNumbers, new TryCounts("1"));
```
저는 이러한 관점도 보여드리고 싶네요! 😄 
위 방식으로 해보시라는것은 결코 아닙니다!

테스트 가능하게 하기 위한 고민 or 의존성을 어디까지 가져갈것인가에 대한 고민 정도로 봐주세요~